templates/debrief.html

```diff
@@ -48,9 +48,11 @@ <h5>Data Options</h5>
             <div id="debrief-graph" class="row svg-container">
                 <button id="graph-switch-technique" class="graph-switch" value="technique" onclick="switchGraphView(this)">Technique</button>
                 <button id="graph-switch-tactic" class="graph-switch" value="tactic" onclick="switchGraphView(this)">Tactic</button>
+                <button id="graph-switch-netpath" class="graph-switch" value="netpath" onclick="switchGraphView(this)">Netpath</button>
```

Comment:
This isn't a big deal, and I agree that "Lateral Movement" might be too long a string for this button, but there might be a clearer name out there.  Maybe "LM Path" with "View lateral movements" as alt-text?

Reply:
opted for "Attack Path" in the most recent commit, thoughts? I was worried LM Path might be too vague since it's an abbreviation.